src/main/java/com/infomaniak/lib/core/auth/CredentialManager.kt

```diff
@@ -97,6 +97,8 @@ abstract class CredentialManager {
                     user = getUserById(userId)
                     return user?.apiToken
                 }
+
+                override fun getCurrentUserId(): Int = userId
```

Comment:
It's not very clear that this part is not important. Because, as is, it will never be different